main.tf

```diff
@@ -1,3 +1,17 @@
+locals {
+  website_config = {
+    redirect_all = [{
+      redirect_all_requests_to = "${var.redirect_all_requests_to}"
+    }]
+
+    default = [{
```

Comment:
thanks @sarkis 
just to confirm, the list with one item (map) works on lines 3 and 7? 
before it was just a single `map`:

```
website {
    index_document = "${var.index_document}"
    error_document = "${var.error_document}"
    routing_rules  = "${var.routing_rules}"
  }
```

Reply:
@aknysh that is correct, terraform is expecting a list here for some reason: https://github.com/terraform-providers/terraform-provider-aws/blob/25b4e92403709e8581cbc8d17e29b6a5b253a04e/aws/resource_aws_s3_bucket.go#L112. 

I originally thought the same as you but was greeted with: 
```
Error: aws_s3_bucket.default: website: should be a list
```